docs/extensions/datacollector-migration.md

```diff
@@ -0,0 +1,44 @@
+# Migrate your TestPlatform v0(TPv0)/TestPlatoformv1(TPv1) datacollector to TestPlatform v2(current version) datacollector
+DataCollectors were supported in TPv0(mstest.exe)/TPv1(vstest.console.exe). 
+This document will walk you through the changes that are required to migrate your TPv0/TPv1 DataCollector to work with TPv2.
+
+## Referencing DataCollector Framework
+Previously, `DataCollector` absract class was present in `Microsoft.VisualStudio.QualityTools.ExecutionCommon.dll` under namespace `Microsoft.VisualStudio.TestTools.Execution`.
+
+Now, `DataCollector`abstract class is present in Object Model. Add reference to `Microsoft.TestPlatform.ObjectModel` (preview) nuget package. DataCollector APIs are present under namespace `Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection`
```

Comment:
Which nuget package version should a user refer for ObjectModel? Link please.

Should they build the datacollector for netstandard or for net46? If this question is answered in another doc, please link to it.